Week2/part1.js

```diff
@@ -0,0 +1,68 @@
+const mysql = require("mysql");
+const util = require("util");
+const con = mysql.createConnection({
+  host: "localhost",
+  user: "hyfuser",
+  password: "hyfpassword",
+  database: "new_world"
+});
+
+const execQuery = util.promisify(con.query.bind(con));
+
+async function queryPromise() {
+
+  function capitalOfCountry(x) {
+    return `SELECT country.name, city.name FROM city, country WHERE id = capital AND country.name = ${x};`;
```

Comment:
I really like your separation with functions. =)

But, for security reasons, you should avoid concatenating input strings directly into the query like you did. Specially if the input comes directly from the user. In your case they are all hardcoded so it is kind of fine, but be aware of this. =)

Try one of the following approaches:

```
connection.query('SELECT * FROM users WHERE id = ?', [userId], function (error, results, fields) {
  if (error) throw error;
  // ...
});
```

or 

```
var userId = 'some user provided value';
var sql    = 'SELECT * FROM users WHERE id = ' + connection.escape(userId);
connection.query(sql, function (error, results, fields) {
  if (error) throw error;
  // ...
});
```

Or take a look at this link. 
https://github.com/mysqljs/mysql#escaping-query-values